src/test/java/com/gdsc/studyjamapi/StudyJamApiApplicationTests.java

```diff
@@ -10,4 +10,6 @@ class StudyJamApiApplicationTests {
   void contextLoads() {
   }
 
+
+
 }
```

Comment:
Please don't change something like this. It will increase the number of file changes but give no values.

Reply:
Thank you, I will fix this.